crates/vercel_runtime_macro/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+use glob::glob;
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::AttributeArgs;
+use vercel_runtime_router::{Route, Router};
+
+use std::collections::HashMap;
+use syn::parse_macro_input;
+
+#[proc_macro_attribute]
+pub fn include_api(args: TokenStream, stream: TokenStream) -> TokenStream {
+    let args = parse_macro_input!(args as AttributeArgs);
+    let mut args_map: HashMap<String, String> = HashMap::new();
+
+    args.iter().for_each(|arg| {
+        if let syn::NestedMeta::Meta(syn::Meta::NameValue(syn::MetaNameValue {
+            path,
+            lit: syn::Lit::Str(lit_str),
+            ..
+        })) = arg
+        {
+            if let Some(key) = path.get_ident() {
+                args_map.insert(key.to_string(), lit_str.value());
+            }
+        }
+    });
+
+    let prefix = std::env::var("VERCEL_RUST_EXPERIMENTAL_MACRO_PREFIX").unwrap_or("".to_string());
```

Comment:
I wonder if there's a way to get the part of the workspace this is in, so no prefix is required.

Reply:
This is a tricky one because there is no stable way to detect in which file the macro is consumed.

The `proc_macro_span` thing is the closest I've come to determine the file path directly. However this would require us to use the nightly channel for all builds.

https://stackoverflow.com/questions/60738538/is-there-a-way-to-get-the-file-and-the-module-path-of-where-a-procedural-macro-i